classes/Media.php

```diff
@@ -24,6 +24,8 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
+use JSMin\JSMin;
```

Comment:
Funny as it doesn't crash and no one tells us it can crash :sweat_smile: 
Seems it's never used!